ow_faults/src/FaultInjector.cpp

```diff
@@ -288,6 +308,12 @@ void FaultInjector::checkAntFaults(){
                 m_faults.ant_tilt_encoder_failure || m_faults.ant_tilt_effort_failure);
 }
 
+void FaultInjector::checkPowerFaults(){
+  m_powerFault =  ( m_faults.low_state_of_charge_power_failure || 
+                    m_faults.instantaneous_capacity_loss_power_failure ||
+                    m_faults.thermal_power_failure);
+}
+
```

Comment:
I think what I was hoping for is something similar to what you had here in this no longer referenced method.
Then you would simply publish the **m_powerFault**. The challenge is however, you need to listen to at least two different topics to conclude that there is an **m_powerFault**.

Reply:
What you could probably do is set the variables `low_state_of_charge_power_failure` and `thermal_power_failure` each in its own listener and then produce and publish the value of m_powerFault based on that.

Reply:
Since this function (and the m_powerFault member) are not referenced, they should be removed, but I agree with @Samahu  that they appeared to be useful and have the right logic.

Reply:
removed !